README.md

```diff
@@ -8,24 +8,65 @@ ospool:
+	$ condor_q -hold
+	ap40.uw.osg-htc.org : <128.105.68.62:9618?... @ 08/01/24 15:01:50
+ 	ID         OWNER          HELD_SINCE  HOLD_REASON
+	130.0       alice           8/1  14:56 Transfer input files failure at ⋯
 
-	$ condor_q -better-analyze JOB-ID -pool POOL-NAME
+In this particular case, a user had this in his or her HTCondor submit file:
+
+	transfer_output_files = outputfile
+
+However, when the job executed, it went into Held state. To see more about the error message `condor_q -hold JOB-ID` can be used. An example error message in this scenario is shown below:
+
+	$ condor_q -hold JOB-ID
+
+	Transfer output files failure at access point… while receiving files from the execution point. Details: Error from ….execute point … failed to send file(s) at apxx; failed to read from file /path: (errno 2) No such file or directory
+
+The reason why the file transfer failed is because `outputfile` was never created on the worker node. Remember that at the beginning we said that the user specifically requested `transfer_outputfiles = outputfile`! HTCondor could not complete this request, and so the job went into Held state instead of finishing normally.
+
+It's quite possible that the error was simply transient, and if we retry, the job will succeed. We can re-queue a job that is in Held state by using `condor_release`: 
+
+	condor_release JOB-ID 
+## Job completed but was unsuccessful
+Under this case two things will be considered. First, your job has run but the code did not execute correctly or in the expected manner. Second, your job ran but it did not produce/transferred the desired output files.
+
+For these secnarios having various `ls` or `echo` statemets that shows the contents of the directory and code's progression is helpful in diagnosing the actual issue. An example is shown here:
+
+	echo "Here are the files at the start of the job:"
+	ls -R
+	echo "Executing main command..."
+	my_command	#Part of the original code
+	echo "... finished. Here are the files at the end of the job:"
+	ls -R
+
+For your commands if eanbling debugging or verbose logging is an option then do those. Moreover, [condor_chirp](https://www.google.com/url?q=https://htcondor.readthedocs.io/en/latest/man-pages/condor_chirp.html&sa=D&source=editors&ust=1740162417565289&usg=AOvVaw1HJ8p454psoN6alPpujatn) can be a very useful tool in this regard as it sends information directly back to the Access Point. For example, the following will add a statement to your .log file:
+	command1
+	condor_chirp ulog "Finished executing command1"
+	command2
+## Job does not start
+
+Matchmaking cycle can take more than 5 minutes to complete and it can be longer if the server is busy. Specially, the more/more-specific resources are requested, the longer is the wait. For example if you are submitting a lot of GPU jobs, there may not be enough `GPUs` in the pool. This type of issues can be diagnosed using the `-better-analyze` flag 
```

Comment:
"Matchmaking" is a little jargon-y. Can you explain a little more what is meant by that here?